bot.py

```diff
@@ -363,24 +388,26 @@ async def openlab(ctx):
 async def torch(ctx):
```

Comment:
The point of the `.torch` secret command was to memorialize '21 officer board 😅

May I suggest moving the revised contents to a `.officers` command instead? You'll have to also add that to the `.help` output.